netkan/netkan/notifications.py

```diff
@@ -23,7 +23,14 @@ def emit(self, record):
         self.webhook.send(f'```{fmt}```' if "\n" in fmt else fmt)
 
 
-def setup_log_handler():
+def setup_log_handler(debug=False):
+    if not sys.argv[0].endswith('gunicorn'):
+        level = logging.DEBUG if debug else logging.INFO
+        logging.basicConfig(
+            format='[%(asctime)s] [%(levelname)-8s] %(message)s', level=level
+        )
+        logging.info('Logging started for \'%s\' at log level %s', sys.argv[1], level)
```

Comment:
This is giving wrong output already in your sample command, this should say `Logging started for 'indexer'`:

```
[2019-11-11 09:14:41,812] [INFO    ] Logging started for '--debug' at log level 10
```

Reply:
Maybe we don't need to repeat back the command name to the user, and this could just state the logging level?

Reply:
When you're running on ECS, it's pretty easy to pick the wrong container.
```
$ docker logs -f f341ec0a3918
# github.com:22 SSH-2.0-babeld-2e9d163d
```
It's something that will stand out if you picked the wrong one.